bitcoin/shadouble.c

```diff
@@ -15,9 +15,4 @@ void sha256_double(struct sha256_double *shadouble, const void *p, size_t len)
 	sha256(&shadouble->sha, &shadouble->sha, sizeof(shadouble->sha));
 }
 
-void sha256_double_done(struct sha256_ctx *shactx, struct sha256_double *res)
```

Comment:
I still think that this function should be reverted to upstream rather than removed. If upstream will start using it in next release, code will break.

Reply:
https://github.com/Groestlcoin/lightning/tree/sha256_done

Reply:
> I still think that this function should be reverted to upstream rather than removed. If upstream will start using it in next release, code will break.

@yura-pakhuchiy some minor private info here, I am highly autistic, and my social skills are very limited, is this a question or an answer ?

Reply:
It was a suggestion which @gruve-p implemented in
https://github.com/Groestlcoin/lightning/commit/ebbe575003e6d5d7344db8fe9a0dd746d0a1c71e

Reply:
So that's how boss "hold my beer" thumbs up are done, Thanks a lot, for me it's so enlightning to be able to work with real humans, I learn every day more and more. One day I will also be a real human, what do you think ? Do I make progress ?